docs/exts/sphinx_script_update.py

```diff
@@ -101,11 +101,22 @@ def build_finished(app, exception):
     script_url = app.config.redoc_script_url
     output_filename = "script.js"
 
+    fetch_and_cache(script_url, output_filename)
+
+
+def build_finished(app, exception):
```

Comment:
I download the necessary files at the start of the documentation build process to avoid a problem with snowflake-connector-python.
See: https://github.com/snowflakedb/snowflake-connector-python/issues/324